main.go

```diff
@@ -57,6 +60,11 @@ type ProjectsResponse struct {
 	Projects []Project `json:"data"`
 }
 
+func NewClient(APIKey string) Client {
+	client := &http.Client{}
+	return Client{APIKey: APIKey, client: client}
```

Comment:
Minor thing: there's no need to assign the `http.Client` pointer to a variable. You can just include it inline in the `Client` struct.